crates/rpc-types/src/eth/filter.rs

```diff
@@ -356,6 +356,14 @@ impl Filter {
         self
     }
 
+    /// Return `true` if filter configured to match pending block
```

Comment:
this should mention that this is the case if both from and to block tags are set to pending